src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -30,7 +50,7 @@ export const FilterSidebar = () => {
       position={{ sm: 'fixed', md: 'sticky' }}
       top={{ sm: '40', md: 'unset' }}
       left={{ sm: '0', md: 'unset' }}
-      width={{ sm: 'full', md: 'auto' }}
+      width={{ sm: 'full', md: '332', lg: '332' }}
```

Comment:
the search container shrinks when typing text.  This isn't part of my task, just a solution to improve the experience for now